test/gtest/common/mem_buffer.cc

```diff
@@ -32,6 +32,10 @@
 
 #endif
 
+#if HAVE_NVML_H
```

Comment:
Need to add this definition to `configure`?

Reply:
it is already defined and coming from config.h